### PR TITLE
Improve sending feedback

### LINKED ISSUE #1820: Migrate FeedbackJobIntentService to WorkManager or use another approach

### PR DESCRIPTION
**Pull request by DenBond7**
This PR improved sending feedback.

close #1820 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)
- Difficult to test (explain why)
- Not worth testing
- Tests will be added later (issue #...)
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities

### CODE REVIEW

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/PostHelpFeedbackResponse.kt [hunk 1]
```diff
@@ -52,7 +52,7 @@ data class PostHelpFeedbackResponse constructor(
   override fun writeToParcel(dest: Parcel, flags: Int) =
     with(dest) {
       writeParcelable(apiError, 0)
-      writeInt((if (isSent) 1 else 0))
+      writeInt((if (isSent == true) 1 else 0))
```
Comment (IvanPizhenko) [inline FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/PostHelpFeedbackResponse.kt:55]: shouldn't we get into account that `isSent` can be null and serialize something different for `null`?

File: FlowCrypt/src/main/java/com/flowcrypt/email/model/Screenshot.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  override fun equals(other: Any?): Boolean {
+    if (this === other) return true
+    if (javaClass != other?.javaClass) return false
+
+    other as Screenshot
+
+    if (!byteArray.contentEquals(other.byteArray)) return false
+
+    return true
```
Comment (IvanPizhenko) [inline FlowCrypt/src/main/java/com/flowcrypt/email/model/Screenshot.kt:28]: last 3 non-empty lines of code can be just `return byteArray.contentEquals((other as Screenshot).byteArray)`

**Review by IvanPizhenko — APPROVED** [review]
Looks good in general, couple of comments.

File: FlowCrypt/src/main/java/com/flowcrypt/email/model/Screenshot.kt (hunk 2 above)
Comment (DenBond7) [reply FlowCrypt/src/main/java/com/flowcrypt/email/model/Screenshot.kt:28]: Agree. But this code was generated by Android Studio. I would like to leave it as-is. Later this class can be changed and modification will be simpler if we leave it as-is.

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/PostHelpFeedbackResponse.kt (hunk 1 above)
Comment (DenBond7) [reply FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/api/PostHelpFeedbackResponse.kt:55]: Good one. I will create a separate issue for that.

**Pull request merged**